pdns/recursordist/rec_metrics.hh

```diff
@@ -83,6 +84,9 @@ public:
     case PrometheusMetricType::histogram:
       return "histogram";
       break;
+    case PrometheusMetricType::multicounter:
+      return "multicounter";
```

Comment:
I wonder if returning "counter" here would not make the code in `prometheusMetrics()` simpler, but that does not matter much.

Reply:
I'll take a look at that when finishing #10605 (which depends on this PR)